README.md

```diff
@@ -39,7 +39,3 @@ Give a ⭐️ if you like this project!
 - Hat tip to anyone whose code was used
 - Inspiration
 - etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly add the license to your Readme file as the previous reviewer requested.